packages/CMakeLists.txt

```diff
@@ -22,9 +22,8 @@ configure_file(${PROJECT_SOURCE_DIR}/packages/debian.${CPP_PACKAGE}.install
                ${PROJECT_BINARY_DIR}/build-packages/debian.${CPP_PACKAGE}.install COPYONLY)
 configure_file(${PROJECT_SOURCE_DIR}/packages/debian.${CPP_PACKAGE}-dev.install
                ${PROJECT_BINARY_DIR}/build-packages/debian.${CPP_PACKAGE}-dev.install COPYONLY)
-# no python package for Debian because there is only SWIG 3.10 on Debian 9 :-/
-#configure_file(${PROJECT_SOURCE_DIR}/packages/debian.${PYTHON_PACKAGE}.install
-#               ${PROJECT_BINARY_DIR}/build-packages/debian.${PYTHON_PACKAGE}.install COPYONLY)
+configure_file(${PROJECT_SOURCE_DIR}/packages/debian.${PYTHON_PACKAGE}.install
+               ${PROJECT_BINARY_DIR}/build-packages/debian.${PYTHON_PACKAGE}.install COPYONLY)
```

Comment:
Is everything required (mostly CFFI I guess) available on all supported Debians and Debian-based distributions? Debian 9 and Ubuntu 16.04 are the oldest supported.

Reply:
Yes I guess CFFI has been present in debian/ubuntu for a long time.

https://packages.debian.org/stretch/python3-cffi
https://packages.ubuntu.com/xenial/python3-cffi

The version is a bit old but it "should" work.

Reply:
Okay, just checking.